srcpkgs/lightdm-slick-greeter/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'lightdm-slick-greeter'
+pkgname=lightdm-slick-greeter
+version=1.4.1
+revision=1
+wrksrc="slick-greeter-${version}"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --sbindir=/usr/bin --libexecdir=/usr/lib/lightdm"
+hostmakedepends="autoconf automake pkg-config intltool gettext-devel glib-devel vala-devel"
+makedepends="lightdm-devel libxklavier-devel gtk+3-devel libcanberra-devel"
+short_desc="Lightdm Slick Greeter"
+maintainer="chgt <christian.gimbatti@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/linuxmint/slick-greeter"
+distfiles="${homepage}/archive/${version}.tar.gz"
```

Comment:
Please repeat the URL instead of using `${homepage}`. The listing of [void updates](https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt) is otherwise less helpful.